LICENSE.txt

```diff
@@ -1,6 +1,12 @@
+Scarpe, Lacci and Scarpe-Components code is available under the MIT license.
+
+Bootswatch themes are made by Thomas Park and are released under MIT license.
+Bootstrap Icons are by Bootstrap Icons and are released under MIT license.
+
+=================
 The MIT License (MIT)
 
-Copyright (c) 2022 TODO: Write your name
+Copyright (c) 2022-present, Scarpe team and contributors.
```

Comment:
For copyright, either way is basically fine. It would get less fine if there was some kind of a copyright fight about who could do what. But given that I never expect any level of commercial success, and the MIT license is really permissive, we're probably fine here.

We'd need to be a lot more careful if this copyright would ever be a thing we could usefully *sell*.